metis-enrichment/metis-enrichment-client/src/main/java/eu/europeana/enrichment/rest/client/dereference/DereferencerImpl.java

```diff
@@ -249,6 +250,8 @@ public DereferencedEntities dereferenceOwnEntities(Set<ReferenceTerm> resourceId
                    result.putIfAbsent(notFoundOwnId, Collections.emptyList());
                  });
       return new DereferencedEntities(result, reports, classType);
+    } catch (CancellationException e){
```

Comment:
This looks reasonable to me, although I would add a LOGGER.warn when this happens as well.
Applies for all locations.